Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/ProjectSettings/UnityAdsSettings.asset

```diff
@@ -0,0 +1,11 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!292 &1
+UnityAdsSettings:
```

Comment:
Jeff, is this actually working? Having ProjectSettings sinside Assets folder?

Do you need your own Project Settings? Should this not just live in it own folder as a Scene in the same project?